### PR TITLE
Import ionicons

### PR DESCRIPTION
**Pull request by akshara-sun**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -59,5 +59,14 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
+    <ion-icon name="search-sharp"></ion-icon>
```
Comment (amilajack) [inline public/index.html:62]: The icons should be added in JSX

**Review by amilajack — CHANGES_REQUESTED** [review]

File: public/index.html (hunk 1 above)
Comment (amilajack) [reply public/index.html:62]: ionicons are web components. See https://developers.google.com/web/fundamentals/web-components/customelements.

**Review by amilajack — CHANGES_REQUESTED** [review]
Please add `fixes #...` to PR description

**Review by hamirmahal — CHANGES_REQUESTED** [review]
Should we ignore .DS_Store files? I believe all they do is tell macOS how to arrange files in Finder.

We can add it to the repository's [`.gitignore`](https://github.com/amilajack/freesound-player/blob/master/.gitignore) file by adding the line

`.DS_Store`

to it.

We can also remove .DS_Store files from this commit to prevent them from appearing in the repository.

File: src/App.js [hunk 2]
```diff
@@ -5,6 +5,7 @@ import Sound from "./Sound";
 import Search from "./Search";
 import auth, { init } from "./Auth";
 
+
```
Comment (amilajack) [inline src/App.js:8]: remove this space

**Review by amilajack — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 2 above)
Comment (akshara-sun) [reply src/App.js]: Just pushed new changes.

Comment (akshara-sun) [reply src/App.js]: I'll close the issue once these changes are reviewed and merged to master.

**Review by amilajack — APPROVED** [review]

**Comment by amilajack** [comment]
@hamirmahal I agree. We should ignore `.DS_Store` in a followup PR

**Pull request merged**